Packs/HYASInsight/ReleaseNotes/1_1_0.md

```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### HYAS Insight
+
+- Added new commands for **hash** type.
```

Comment:
Please list the new commands, see example [here](https://github.com/demisto/content/blob/master/Packs/AzureSentinel/ReleaseNotes/1_5_0.md?plain=1#L32).